app/presenters/government_result.rb

```diff
@@ -8,7 +8,9 @@ def to_hash
       metadata_any?: metadata.any?,
       sections: sections,
       sections_present?: sections.present?,
-      government: true
+      government: true,
+      historic?: historic?,
+      government_name: government_name
```

Comment:
Should probably add a trailing comma on this line to conform to style for cleaner diffs in the future.